deploy-web/src/pages/providers/[owner]/index.tsx

```diff
@@ -60,7 +58,7 @@ const ProviderDetailPage: React.FunctionComponent<Props> = ({ owner, _provider }
     isLoading: isLoadingStatus,
     refetch: getProviderStatus
   } = useProviderStatus(provider?.hostUri, {
-    enabled: false,
+    enabled: true,
```

Comment:
It was not working properly? I don't remember exactly why I made explicity query only once before. It just made me think, at some point the `/status` rest endpoint won't exist anymore, so we'll probably have to migrate to the grpc in the frontend as well.

Reply:
It was working when coming to the provider page from another page, but it was not loading when opening/refreshing directly on the provider page. I think you may have put it to false so that it doesnt query the endpoint during server rendering, didn't look too far into it :o

Reply:
I don't think it was for ssr, but more to not spam the provider endpoint since the status doesn't really change.

Reply:
Got it, yeah for sure we could change react-query options to keep the cache longer for `/version`. I'll let it like this for now, but we will need to refactor the provider pages after the rebrand. This merging of 3 sources into the same provider object is causing weird issues. Also just realized a similar problem happens with the "Your active leases" count.